lib/matplotlib/tests/test_backend_qt.py

```diff
@@ -166,24 +166,49 @@ def custom_handler(signum, frame):
 
 
 @pytest.mark.parametrize(
-    'qt_key, qt_mods, answer',
+    "qt_key, qt_mods, answer",
     [
-        ('Key_A', ['ShiftModifier'], 'A'),
-        ('Key_A', [], 'a'),
-        ('Key_A', ['ControlModifier'], 'ctrl+a'),
-        ('Key_Aacute', ['ShiftModifier'],
-         '\N{LATIN CAPITAL LETTER A WITH ACUTE}'),
-        ('Key_Aacute', [],
-         '\N{LATIN SMALL LETTER A WITH ACUTE}'),
-        ('Key_Control', ['AltModifier'], 'alt+control'),
```

Comment:
I'm not sure why this is "control" rather than "ctrl" here, but it appears to work either way for me locally, and this would get you an easier string replacement.